### ISSUE TITLE
Desktop app size is 161 Mb instead 90 Mb

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#8014]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___
**Issue found when executing #7744**
## Action Performed:
1. Download the Desktop app .dmg and verify the size

## Expected Result:
The size is close to 90 Mb

## Actual Result:
The size .dmg is 161 Mb instead 90 Mb.

## Workaround:
Unknown
## Platform:

Where is this issue occurring?

- Desktop App

**Version Number:**  v1.1.41-4
**Reproducible in staging?:**  Y
**Reproducible in production?:** Y
**Email or phone of affected tester (no customers):** 
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** 
![Bug5478811_Screen_Shot_2022-03-04_at_4 28 02_PM](https://user-images.githubusercontent.com/43996225/156859459-9d44abb9-32d4-45a0-be1a-1413ccaa2614.png)

**Expensify/Expensify Issue URL:** 
**Issue reported by:** Applause
**Slack conversation:** 

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by MelvinBot** [comment]
Triggered auto assignment to @Beamanator (`Engineering`), see https://stackoverflow.com/c/expensify/questions/4319 for more details.

**Comment by Beamanator** [comment]
@kidroca it looks like in the test steps of your PR (https://github.com/Expensify/App/pull/7744), you mentioned:

> Download the Desktop app .dmg and verify the size is close to 90 Mb instead of 334Mb

However, when downloading I see the app as 154mb - do you think this is "close enough" to 90mb or is something else included that needs to be removed?

**Comment by francoisl** [comment]
154 MB doesn't sound close enough to 90MB.

I'm thinking this might have had to do with https://github.com/Expensify/App/issues/7987, which is now resolved. When I download the staging app now, its size is 91MB as expected, but it shows as an unsupported app and I can't install it at all.

<img width="545" alt="image" src="https://user-images.githubusercontent.com/2229301/157111570-8f27e936-778f-4469-870b-ed9b7029de02.png">

**Comment by Beamanator** [comment]
Yeah good call that does look related - @roryabraham can you help with this issue?

**Comment by Beamanator** [comment]
@francoisl which download link are you using for the staging app? I used https://staging.new.expensify.com/NewExpensify.dmg and I'm currently seeing a 154 MB app downloading - i'll try running it once it downloads

**Comment by kidroca** [comment]
@Beamanator 
Sorry the requirements should have been updated 

We had some problems getting the app on staging and updating after #7744 
- https://github.com/Expensify/App/issues/7987

@roryabraham made some updates to fix the deploy blocker and have things more close to how they used to be 

This resulted in the `universal` app being served at `https://staging.new.expensify.com/NewExpensify.dmg` 
Current we serve only 1 architecture 
- [NewExpensify.dmg](https://staging.new.expensify.com/NewExpensify.dmg) and it happens to be the `universal` app which is 2x size to support both x64 and arm64 (160MB installer, 350MB installed)

But we can also serve:
- NewExpensify-x64.dmg That's the version served in the past. It works on arm64 through Rosetta (90MB installer, 200MB installed)
- NewExpensify-arm64.dmg. This version works natively on M1 macs (90MB installer, 200MB installed)

**Comment by kidroca** [comment]
This is not an issue, or at least it's not a deploy blocker 
The desktop app being 161Mb is still 2x smaller than what we had in the past 
There's a separate ticket handling x64/arm64 distribution 
(searching for a link)

**Comment by kidroca** [comment]
@francoisl 
You should be able (after App is dragged to applications), select get info and check the `Kind` you have installed:
<img width="263" alt="Screenshot 2022-03-07 at 23 32 28" src="https://user-images.githubusercontent.com/12156624/157121274-f18c7ee3-4990-4b33-95b2-d59b60b141b5.png">

I guess M1 macs can run x64, but x64 can't run the "Apple Silicon" version 
While `universal` works on both

**Comment by Beamanator** [comment]
Awesome thanks for the clarification @kidroca 👍 💯 

I'll close this out since, as you mentioned above, the larger size is expected, and we have another ticket discussion possible options [here](https://github.com/Expensify/App/issues/8014#issuecomment-1061154627)

**Comment by francoisl** [comment]
Ok thanks for the context!
